src/applications/personalization/dashboard/tests/containers/YourApplications.unit.spec.jsx

```diff
@@ -316,20 +322,68 @@ describe('<YourApplications>', () => {
     tree.unmount();
   });
 
-  it('should render a HCAStatusAlert', () => {
-    const tree = shallow(
-      <YourApplications
-        {...defaultProps}
-        shouldRenderContent
-        shouldRenderHCAAlert
-        hcaEnrollmentStatus={enrollmentStatus}
-      />,
-    );
-    const alert = tree.find('HCAStatusAlert');
-    expect(alert.length).to.equal(1);
-    expect(alert.prop('enrollmentStatus')).to.equal(
-      enrollmentStatus.enrollmentStatus,
-    );
-    tree.unmount();
+  describe('if user is verified', () => {
+    let tree;
+    beforeEach(() => {
+      enrollmentStatusSpy = sinon.spy();
+      dismissedNotificationsSpy = sinon.spy();
+      tree = shallow(
+        <YourApplications
+          {...defaultProps}
+          getEnrollmentStatus={enrollmentStatusSpy}
+          getDismissedHCANotification={dismissedNotificationsSpy}
+          shouldRenderContent
+          shouldRenderHCAAlert
+          hcaEnrollmentStatus={enrollmentStatus}
+          profileState={{ verified: true }}
+        />,
+      );
+    });
+    afterEach(() => {
+      tree.unmount();
+    });
+    it('should call the `enrollment_status` endpoint', () => {
+      expect(enrollmentStatusSpy.called).to.be.true;
+    });
+    it('should call the `dismissed_statuses` endpoint', () => {
+      expect(dismissedNotificationsSpy.called).to.be.true;
+    });
+    it('should render a HCAStatusAlert', () => {
+      const alert = tree.find('HCAStatusAlert');
+      expect(alert.length).to.equal(1);
+      expect(alert.prop('enrollmentStatus')).to.equal(
+        enrollmentStatus.enrollmentStatus,
+      );
+    });
+  });
+
+  describe('if user is not verified', () => {
+    let tree;
+    beforeEach(() => {
+      enrollmentStatusSpy = sinon.spy();
+      dismissedNotificationsSpy = sinon.spy();
+      tree = shallow(
+        <YourApplications
+          {...defaultProps}
+          getEnrollmentStatus={enrollmentStatusSpy}
+          getDismissedHCANotification={dismissedNotificationsSpy}
+          shouldRenderContent
+          profileState={{ verified: false }}
+        />,
+      );
+    });
+    afterEach(() => {
+      tree.unmount();
+    });
+    it('should not call the `enrollment_status` endpoint', () => {
+      expect(enrollmentStatusSpy.notCalled).to.be.true;
+    });
+    it('should not call the `dismissed_statuses` endpoint', () => {
+      expect(dismissedNotificationsSpy.notCalled).to.be.true;
+    });
+    it('should not render a HCAStatusAlert', () => {
```

Comment:
What is rendered when the user is not verified? Nothing? Some other message? Might be worth testing that.

Reply:
it will still render alerts for saved-in-progress forms if there are any. will add a test for that.